### DISCUSSION TITLE
Thank you for your contributions (October 2023)

### DISCUSSION THREAD

**Discussion by MisRob** [learningequality/kolibri#11503] [Community]
During October, we received amazing contributions from

@ShivangRawat30, @haldaranup, @andreamisuraca, @thesujai, @pwithnall, @FireSuperior482, @muditchoudhary, @Jaspreet-singh-1032, @a6ar55, @im-NL, @BabyElias, @KshitijThareja, and also our good friend Devon @indirectlylit.

Thanks to each one of you! We've mentioned all of you in our newest [Community Newsletter](https://learningequalitynewsletter.getresponsepages.com/).

On this opportunity, I'd like to invite all new contributors to take our open-source community survey (https://github.com/learningequality/kolibri/discussions/11320). Thank you to those who have already did so, we've already gathered some valuable insights. In December, we are planning to start evaluating the survey results in more detail and locate possible improvements in our engagement and processes.

**Comment by 0xV0YD** [comment]
@MisRob hello mam i would like to contribute now again as the time i was off , i learned about django , app dev and now i am ready to indulge fully in open source contribution can i start from setting up the app locally on my pc ?

**Reply by MisRob** [reply]
Hi @yash1378, I linked contributing guidelines in the other thread you posted.
